functions/New-DbaDbFileGroup.ps1

```diff
@@ -0,0 +1,124 @@
+function New-DbaDbFileGroup {
+    <#
+    .SYNOPSIS
+        Creates a new empty filegroup.
+
+    .DESCRIPTION
+        Creates a new filegroup for the specified database(s). The filegroup properties cannot be set on an empty filegroup, so it is necessary to add one or more files and then use the Set-DbaDbFileGroup command to set the filegroup properties.
+
+    .PARAMETER SqlInstance
+        The target SQL Server instance or instances. This can be a collection and receive pipeline input to allow the function
+        to be executed against multiple SQL Server instances.
+
+    .PARAMETER SqlCredential
+        Login to the target instance using alternative credentials. Accepts PowerShell credentials (Get-Credential).
+
+        Windows Authentication, SQL Server Authentication, Active Directory - Password, and Active Directory - Integrated are all supported.
+
+        For MFA support, please use Connect-DbaInstance.
+
+    .PARAMETER Database
+        The target database(s).
+
+    .PARAMETER FileGroupName
```

Comment:
Could you rename this to just FileGroup for all three functions please? We tend to drop the `name`.